code/index.html

```diff
@@ -1,30 +1,162 @@
+          </div>
+          <div class="submit-button">
+            <button type="submit" class="order-button">Order</button>
+          </div>
+        </form>
+      </section>
+      <section class="accordion-container">
+        <h3 class="header-accordion">Most common questions</h3>
+        <div class="accordion-item">
+          <div class="accordion-item-header" id="question1">
+            Where does your coffee come from?
+          </div>
+          <div class="accordion-item-body">
+            <div class="accordion-item-body-content">
+              Our coffee comes from Kenya where Arabica coffee grows on the rich volcanic soil that is found in the
+              highlands of the country. Many people list Kenyan coffee among the five best in the world.
+            </div>
+          </div>
+        </div>
+        <div class="accordion-item">
+          <div class="accordion-item-header" id="question2">
+            What type of your coffee has the most caffeine?
+          </div>
+          <div class="accordion-item-body">
+            <div class="accordion-item-body-content">
+              Highly-caffeinated coffee (e.g. Death Wish Coffee, Biohazard Coffee, etc.)
+            </div>
+          </div>
+        </div>
+        <div class="accordion-item">
+          <div class="accordion-item-header" id="question3">
+            How much ground coffee should I use?
+          </div>
+          <div class="accordion-item-body">
+            <div class="accordion-item-body-content">
+              A general guideline is called the "Golden Ratio" - one to two tablespoons of ground coffee for every six
+              ounces of water.
+            </div>
+          </div>
+        </div>
+        <div class="accordion-item">
+          <div class="accordion-item-header" id="question4">
+            Why is coffee sometimes bitter?
+          </div>
+          <div class="accordion-item-body">
+            <div class="accordion-item-body-content">
+              Good quality coffee will commonly have some bitter elements, but bitterness should not be an overwhelming
+              component.
+            </div>
+          </div>
+        </div>
+        <div class="accordion-item">
+          <div class="accordion-item-header" id="question5">
+            Is your coffee Fairtrade?
+          </div>
+          <div class="accordion-item-body">
+            <div class="accordion-item-body-content">
+              Of course, choosing Fairtrade coffee makes a difference!
+            </div>
+          </div>
+        </div>
+    </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+  </div>
+  <footer>
+    <a target="_blank" href="https://github.com/JuliaNiki">
+      <p class=" footer"> 🐢 Github JuliaNiki ☕️ </p>
+    </a>
+  </footer>
+  <script src="./script.js"></script>
+  <!-- dont forget to link the javascript file here! -->
+</body>
```

Comment:
You have a great way of structuring your code with a nice flow that makes it easy to read and understand. Great job!

Reply:
Thank you so much, feel even more proud after I've read your comments :) 
They made me realise again how much progress has been done.